### ISSUE TITLE
Feedback Request: "Miscreant" minor(ish) antag

### ISSUE THREAD

**Issue by GreatBigFailure** [d3athrow/vgstation13#35262]
An occasional extended round can be fun, but if too common then it's just boring and kills the pop. 
People who understand how the game picks the mode far better than I say it's because a lot of different antags have too specific trigger conditions and so extended's odds increase when conditions aren't met. 

So rather than tinker with it, it's likely more constructive to dilute the pool extended is picked from, but antags aren't easy to make, so:

**Miscreant**

Rather than make an entirely new antag with an entirely new niche, the idea is to recycle a lot of content other antags don't really use, and make their niche a sort of traitor-lite. So something similar to a Survivor, they have antag permissions but aren't specifically told/meant to mess things up too much. 
You know how as an extended round goes on, people start acting more and more like antags until someone finally gets banned? They're meant to be near that "until someone finally gets banned" point from the start. So dangerous contraptions, line-toeing between IC & OOC crime, etc. 

**How to accomplish this**

Miscreants would get a different version of the uplink/spellbook/etc that other antags get, something with points to buy things from. However their uplink:
1.  Starts empty
2. Charges points slowly over time
3. Most important, only has items that are more useful for pranks, interesting gimmicks, or general story/event focused antaggery, rather than murder and round ending. 

Some things might not even be antag items, hell you could put basic metal sheets and such in there since it charges over time, whatever would help the miscreant fulfill their gimmick. 

**WYCI/why even make this issue**

I might make it myself, no promises. I'm rusty and have a bad track record with projects like this, but since it's mostly recycling existing content it would be fairly easy. **I would need suggestions for what could go in the uplink, how prices would be decided, additional mechanics, etc. Also if this is a stupid idea or not, it might be a stupid idea**.

[discussion] [gamemode] [help]

**Comment by west3436** [comment]
\>they have antag permissions but aren't specifically told/meant to mess things up too much

Stop this! Every time variations of this come up there is always debate amongst the playerbase and admins as to what is considered "over the line". If you say murder is over the line, then nothing separates this role from your standard greytide.

tl;dr: this role already exists (mechanic)

**Comment by GreatBigFailure** [comment]
> >they have antag permissions but aren't specifically told/meant to mess things up too much
> 
> Stop this! Every time variations of this come up there is always debate amongst the playerbase and admins as to what is considered "over the line". If you say murder is over the line, then nothing separates this role from your standard greytide.
> 
> tl;dr: this role already exists (mechanic)

That's what stopped me from making it before, it's been something I was thinking about for a while. 
The idea essentially is a super-greytide in a way. Do you see any compromise for the ruling issue though? Like maybe saying no supermatter explosions, no singulo, etc, no insta-shuttle call but dangerous gimmicks are fine otherwise?

**Comment by Eneocho** [comment]
All my 👍🏻s if it comes bundled with the fishing update.

**Comment by wotisjanitor** [comment]
I don't think you need to include that caveat that it's not meant to go over the line. I think the whole concept of an uplink slowly appreciating in TC would encourage people to wait a while before going totally bananas, and the new selection of items can also contribute to this. If the Miscreant wants to toolbox people, let him. But he is rewarded for not going full traitor. Also, another way to accomplish this without setting up a code of conduct for the role is to give it jecties that reflect the tone of the antag.

**Comment by GreatBigFailure** [comment]
@wotisjanitor 

That's a good point actually. Just gotta make sure it's seen as a prankster not a murderer and not supply them for murder.
That could circumvent a lot of ruling issues.

**Comment by Blithering** [comment]
Perhaps they could receive a discount based on the least used uplink items instead of job specific, and conversely a price hike in the most commonly used things. That way they'd be near forced to figure out a use for boozy shotgun shells and syndicate brand soap instead of getting the usual emag.

**Comment by Dacendeth** [comment]
If this is just traitor but zero/slow tc I don't see the point and it just feels like bloat as any antag can already be a prankster if he wants, A comparison of what this feels like to me is if we added "wizard but you cant buy spells only items".

**Comment by GreatBigFailure** [comment]
@Blithering 
I think it would depend on what the list of uplink items looked like. Definitely not against the idea though. With that said if I end up going ahead with this it wouldn't be in the first draft, I'm too rusty to add some kind of logging system for traitor items.

@Dacendeth 
I get what you're saying, but I do think there's something to be said for an antag that everyone knows, including the antag, is inherently less dangerous. The delay on the TP-standin accumulating also pushes them to do their job and/or RP more before forming a whole shtick they're committed to. If the list of uplink items is too short and comes off as more of a gutted traitor than anything, I'll agree with you.  
Is Challenger still in? I'd prefer this over challenger, personally. It's basically the opposite of challenger.

**Comment by wotisjanitor** [comment]
I like how some people don't possess reading comprehension. when the issue literally says "the most important thing is there is a different version of the uplink with different items" it is almost impossible to interpret it as "slow tc traitor"

**Comment by Substance455** [comment]
>uproot 4 floor tiles in front of HoP's
>entire sec team goes into manhunt mode to execute one funny assistant

Stop fucking murdering me when i prank the skeleton crew on extended, and ill show you what a true "Miscreant" is, without the need for c*ding or special roles.

**Comment by GreatBigFailure** [comment]
> I like how some people don't possess reading comprehension. when the issue literally says "the most important thing is there is a different version of the uplink with different items" it is almost impossible to interpret it as "slow tc traitor"

In fairness it will have a lot of traitor items. Stuff like inflatable decoys, camera bugs, evidence falsifier, etc. No guns, killer wheelchair, etc. Probably an emag because it's such a flexible item and useful in a lot of gimmicks, but for a lot more TC equivalent compared to traitor. Though I was thinking they could get a single use emag for cheap? 
Then some things that might not be seen as antag items but could be useful to a gimmick but hard to get: super soaker, a handful of trader items, bluespace crystals. Hell, maybe even just some raw materials like uranium. 
Not going to code a lot of/any new items for this.

**Comment by pedr69** [comment]
make objectives that aren't just "kill X and steal the CMO's hypospray" but think of some good ones, maybe add a "minimise casualties" in there

**Comment by GreatBigFailure** [comment]
> make objectives that aren't just "kill X and steal the CMO's hypospray" but think of some good ones, maybe add a "minimise casualties" in there

For this, or for traitor?

**Comment by Blithering** [comment]
I like the idea of being able to buy otherwise difficult to obtain but not uplink items like super soakers.

**Comment by Eneocho** [comment]
The redshirts fear the greyshit with pacid mix in his super soaker and nothing to lose.

**Comment by Blithering** [comment]
Since this is a role that creates fun without being murdered, would it be a good idea to have clowns summoned from code rainbow be miscreants or roll for it?

**Comment by wotisjanitor** [comment]
Miscreants are allowed to murder though

**Comment by Blithering** [comment]
And so it begins...

> Miscreants are allowed to murder though

**Comment by Eneocho** [comment]
Fuck it, the solution is making a player controlled gremlin mob whose only business is "learning" about technology by tinkering and messing with it (almost always breaking it). Think grinch, but without the bag and able to "develop" some tech (self buffs) the more it learns.

**Comment by wotisjanitor** [comment]
> And so it begins...

Yeah people will abuse a role that lets them murder, but there are good and tangible reasons to not do to this, as you have access to new less-than-lethal uplink loot, and you slowly generate TC for it over the course of time.
